gulpfile.js

```diff
@@ -0,0 +1,33 @@
+
+// used by travis-ci to check the hiera json files
+
+var gulp = require("gulp");
+var jsonlint = require("gulp-jsonlint");
+var expect = require('gulp-expect-file');
+var gutil = require('gulp-util');
+
+// custom reporter that should throw exception on lint fail
+var myFailReporter = function (file) {
```

Comment:
Do we have a doc somewhere that I can refer to about our styleguide?

On 8 May 2015 at 10:55, Simon Gaeremynck <notifications@github.com> wrote:

> In gulpfile.js
> <https://github.com/oaeproject/puppet-hilary/pull/144#discussion_r29928711>
> :
>
> > @@ -0,0 +1,33 @@
> > +
> > +// used by travis-ci to check the hiera json files
> > +
> > +var gulp = require("gulp");
> > +var jsonlint = require("gulp-jsonlint");
> > +var expect = require('gulp-expect-file');
> > +var gutil = require('gulp-util');
> > +
> > +// custom reporter that should throw exception on lint fail
> > +var myFailReporter = function (file) {
>
> Super minor nitpick:
>
> We all try to follow the same styleguide and not have any spaces between
> function and the opening brace.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oaeproject/puppet-hilary/pull/144/files#r29928711>.
>